tls/tls.go

```diff
@@ -0,0 +1,76 @@
+// Package tls provides methods for managing tls configuration for apps.
+package tls
+
+import (
+	"encoding/json"
+	"fmt"
+
+	deis "github.com/deis/controller-sdk-go"
+	"github.com/deis/controller-sdk-go/api"
+)
+
+// Info displays an app's tls config.
+func Info(c *deis.Client, app string) (api.TLS, error) {
+	u := fmt.Sprintf("/v2/apps/%s/tls/", app)
+
+	res, reqErr := c.Request("GET", u, nil)
```

Comment:
Note that while you no longer need the `go1.7` hack, you still need to `defer res.Body.Close()`.